Doc/howto/clinic.rst

```diff
@@ -546,15 +547,15 @@ Let's dive in!
 
         &_Py_ID(new_unique_py_id)
 
-    If it does, you'll have to run ``Tools/scripts/generate_global_objects.py``
+    If it does, you'll have to run :program:`Tools/scripts/generate_global_objects.py`
```

Comment:
Some alternatives:

```suggestion
    If it does, you'll have to run :program:`python3 ./Tools/scripts/generate_global_objects.py`
```

or:

```suggestion
    If it does, you'll have to run :program:`make regen-global-objects`
```

Reply:
I resolved most of these remarks in 971c0584fcb7da8809bd0da854963bdf4f5befcd. Hope that looks ok with you. If not, ping me again.

Reply:
The advantage of using `:source:` is that it creates a link that might be interesting for people that want to take a look at the code before running it, whereas `:program:` doesn't.  The docs for `:program"` also say it's for "The name of an executable program.", without mentioning commands you have to execute (that might include paths and cmdline args).

Another thing to keep in mind is how the output looks like:
* [Tools/clinic/clinic.py](https://github.com/python/cpython/tree/3.12/Tools/clinic/clinic.py) -- with `:source:` it looks like a link
* **Tools/clinic/clinic.py** -- with `:program:` it looks bold
* ``Tools/clinic/clinic.py`` -- with ` ``...`` ` it looks monospaced and with a different background

I'm actually not a big fan of how `:program:` is rendered, since it looks just like emphasis rather than a command you have to run.  `:source:` is good when talking about the script in general since it creates the link and you don't have to copy-paste it as a command, but for cases where you do (like the ``make regen-global-objects``) I'm thinking that a simple ` ``...`` ` might be a better option (since it would be more akin to an expression like ``var = func(args)``).

Reply:
> I'm thinking that a simple `` ``...`` `` might be a better option (since it would be more akin to an expression like `var = func(args)`).

I agree; neither `:program:` nor `:source:` are perfect markups here.

Reply:
Reverted in 47061df9dd03fb10c300f8c67d0d9121948ab79f